tests/Http/Api/ActivityApiTest.php

```diff
@@ -305,4 +305,36 @@ public function testQuantityOnActivityIndex()
             ],
         ]);
     }
+
+    /**
+     * Test for retrieving a user's activity with northstar_id and campaign_run_id query params.
+     * This test is to ensure we are not breaking the Gambit flow when it checks to see if a signup exists.
+     *
+     * GET /activity?filter[]=
+     * @return void
+     */
+    public function testActivityIndexWithNorthstarIdAndCampaignRunIdFilters()
```

Comment:
hmm I think we are deprecating this, we can keep this test so this is not a comment blocking merging. But lets not invest anymore in support for this endpoint. We can tell devs they can start using the more RESTful `/v3` endpoints